test/index.js

```diff
@@ -1338,3 +1338,31 @@ describe('parse', function() {
   });
 
 });
+
+describe('regex', function() {
```

Comment:
Let's change the heading to `'regexp'` to match the section heading in the source code.